samples/dotnetcore/Startup.cs

```diff
@@ -0,0 +1,140 @@
+namespace OidcSample
+{
+    using System.Linq;
+    using System.Text;
+    using System.Text.Encodings.Web;
+    using Microsoft.AspNetCore.Authentication;
+    using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            this.Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddAuthentication(options =>
+            {
+                options.DefaultAuthenticateScheme = OpenIdConnectDefaults.AuthenticationScheme;
+                options.DefaultSignInScheme = OpenIdConnectDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = OpenIdConnectDefaults.AuthenticationScheme;
+                options.DefaultScheme = OpenIdConnectDefaults.AuthenticationScheme;
+            })
+            .AddCookie(OpenIdConnectDefaults.AuthenticationScheme, options =>
+            {
+                options.Cookie.SameSite = SameSiteMode.None;
+            })
+            .AddOpenIdConnect("oidc", options =>
+            {
+                options.Authority = "https://api.mpin.io";
```

Comment:
I use it only here and don't see a reason to extract it as a constant. Just looked at other samples - it's the same there too

Reply:
The other samples have the `ISSUER` env var and take that value from there or if the env var is empty, they default to `https://api.mpin.io`.
You're also right that maybe there is no need to configure that, but I did them that way, since hard coding stuff is a bit of an anti pattern. What ever we decide we should generalize in all samples.

Reply:
It doesn't really matter that it is used only here.

Reply:
done